app/src/main/java/kr/co/softcampus/sopt_assignment1/DetailActivity.kt

```diff
@@ -0,0 +1,25 @@
+package kr.co.softcampus.sopt_assignment1
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import kr.co.softcampus.sopt_assignment1.databinding.ActivityDetailBinding
+
+class DetailActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityDetailBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityDetailBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        initData()
+    }
+
+    private fun initData(){
+        binding.ivPhoto.setImageResource(intent.getIntExtra("img", 0))
+        binding.tvName2.setText(intent.getStringExtra("name"))
+        binding.tvDetail2.setText(intent.getStringExtra("detail"))
```

Comment:
스코프 함수를 이용해서 이런식으로 간결하게 표현할 수 있고 또 코틀린 프로퍼티 속성을 이용해서 getter나 setter를 이용하지않고`tvName2.text = intent.getStringExtra("name")` 이런식으로 간결하게도 표현할 수 있습니당 :)
```suggestion
        with(binding) {
        ivPhoto.setImageResource(intent.getIntExtra("img", 0))
        tvName2.text = intent.getStringExtra("name"))
        tvDetail2.text = intent.getStringExtra("detail"))
        }
```